README.md

```diff
@@ -10,8 +10,11 @@
 - HTML
 - CSS
 
+## Live Demo
 
 
```

Comment:
Note: *Please, this change is for your contributing header section of your readme file*

- Please, kindly update the link to the issue page to be that of the current project as the link now is for the "Hello Microverse" project to make your readme more professional.